### PR TITLE
Issue 2000 - Plug'n'hack Add-on Title Caps

### PR DESCRIPTION
**Pull request by kingthorin**
Adjust strings in Messages.properties.
Update version and changes in manifest. (https://github.com/zaproxy/zap-extensions/releases)

There seem to be more for PnH, but I didn't want to get too carried away when I wasn't sure of all the places in the UI that they appeared (so that I can ensure I'm making the right adjustment(s)).

### CODE REVIEW

File: src/org/zaproxy/zap/extension/plugnhack/ZapAddOn.xml [hunk 1]
```diff
@@ -1,13 +1,13 @@
 <zapaddon>
 	<name>Plug-n-Hack Configuration</name>
-	<version>12</version>
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/plugnhack/ZapAddOn.xml]: It was not yet released.

Comment (kingthorin) [reply src/org/zaproxy/zap/extension/plugnhack/ZapAddOn.xml]: Hmmmmm......ok.

Comment (kingthorin) [reply src/org/zaproxy/zap/extension/plugnhack/ZapAddOn.xml]: Was "Update for 2.7.0." from v11?

Comment (thc202) [reply src/org/zaproxy/zap/extension/plugnhack/ZapAddOn.xml]: No, that was done in v12, #1212.

Comment (kingthorin) [reply src/org/zaproxy/zap/extension/plugnhack/ZapAddOn.xml]: Ok, will adjust accordingly.

**Pull request merged**